internal/provisioner/helm_utils.go

```diff
@@ -94,7 +94,7 @@ func (d *helmDeployment) Exists() (bool, error) {
 	}
 
 	for _, chart := range list.asSlice() {
-		if chart.Name == d.chartDeploymentName && chart.Chart == d.chartName && chart.Namespace == d.namespace {
```

Comment:
What was the problem with this check? just curious.

Reply:
They are different entities. `d.chartName` holds `ingress-nginx/ingress-nginx` but `chart.Chart` holds `ingress-nginx-4.2.0`